homeassistant/components/yeelight/light.py

```diff
@@ -58,9 +63,15 @@
 SUPPORT_YEELIGHT_RGB = SUPPORT_YEELIGHT_WHITE_TEMP | SUPPORT_COLOR
 
 ATTR_MODE = "mode"
+ATTR_MINUTES = "minutes"
 
 SERVICE_SET_MODE = "set_mode"
 SERVICE_START_FLOW = "start_flow"
+SERVICE_SET_COLOR_SCENE = "set_color_scene"
+SERVICE_SET_HSV_SCENE = "set_hsv_scene"
+SERVICE_SET_CT_SCENE = "set_color_temp_scene"
+SERVICE_SET_CF_SCENE = "set_color_flow_scene"
```

Comment:
It's very easy to misread these constant names with only one character difference and misplace them. Can we write out flow vs temp instead?

Reply:
Fixed